less/mixins.less

```diff
@@ -1 +1,20 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+@mobile: ~'(max-width: 500px)';
+@tablet: ~'(max-width: 800px)';
+@desktop: ~'(max-width: 1500px)';
+
+.flex(@justify, @align) {
```

Comment:
Fun fact: You can give these parameters default values, like
```
.flex(@justify: space-between, @align: center) {
```

That way, whenever you call the mixin later, if you call it like `.flex();` then space-between and center will automatically be applied. Not necessary, but it's a kinda cool feature of mixins.